cognite/client/data_classes/contextualization.py

```diff
@@ -68,10 +84,10 @@ def wait_for_completion(self, timeout=None, interval=1):
         start = time.time()
         while timeout is None or time.time() < start + timeout:
             self.update_status()
-            if self.status not in ["Queued", "Running"]:
+            if self.status not in JobStatus.NOT_FINISHED_STATUSES:
```

Comment:
With above suggestion:
```suggestion
            if not self.status.is_finished():
```